converters/toZigbee.js

```diff
@@ -988,7 +989,7 @@ const converters = {
               *
               * Additionally when we get a get payload, only request the fields included.
              */
-            const attributes = [];
+            const attributes = ['colorMode'];
 
             if (!meta.message.color || (typeof meta.message.color === 'object' && meta.message.color.hasOwnProperty('x'))) {
```

Comment:
@Koenkk do we still need this logic? As we're now always reading everything?

Reply:
What do you mean with "always reading everything"? We only read what is requested?

Reply:
Given the chance requested earlier to read colorMode so we could properly do the sync, …

only ready colorMode and say currentlX is not going to do what the user expected, unless our colorMode is xy.